RELEASE_SUPPORT.md

```diff
@@ -4,7 +4,7 @@ Please, refer to the [master branch](https://github.com/eProsima/Fast-DDS/blob/m
 
 *eProsima Fast DDS* maintains several releases with different support cycles.
 Major releases are scarce and they are made when public API is broken (more information about *eProsima Fast DDS* versioning can be found [here](https://github.com/eProsima/Fast-DDS/blob/master/VERSIONING.md)).
-Latest major release is v2.0.0 (June 2020), which introduced the DDS compliant API.
+Latest major release is v3.0.0 (August 2024), which introduced the DDS-XTypes 1.3.
```

Comment:
We may also take the chance to put a link to the `UPGRADING.md`. I am thinking that we should also point to the upgrading file in the release notes for v3.0.0.